src/cmd/nsfs.js

```diff
@@ -243,7 +244,7 @@ async function main(argv = minimist(process.argv.slice(2))) {
         const https_port = Number(argv.https_port) || Number(process.env.ENDPOINT_SSL_PORT) || 6443;
         const https_port_sts = Number(argv.https_port_sts) || -1;
         const metrics_port = Number(argv.metrics_port) || -1;
-        const forks = Number(argv.forks);
+        const forks = Number(argv.forks) || null;
```

Comment:
I don't really like this Number || null thing...
Why can't we use something like? 
```suggestion
        const forks = Number(argv.forks) || config.ENDPOINT_FORKS;
```
and maybe remove what's in line 89...

Reply:
I added your suggestion but didn't remove line 89 because we need it on other flows